build/ci/run_olm_verification.sh

```diff
@@ -16,5 +16,13 @@
 # limitations under the License.
 #
 
-docker build -f build/ci/Dockerfile.courier -t operator-courier .
-docker run --rm -t operator-courier
+docker build -f build/ci/Dockerfile.olm-verification -t operator-olm-verification .
+
+ARCH=$(uname -m)
+
+if [ "${ARCH}" != "ppc64le" ]; then
+  DOCKER_PATH=$(which docker)
+  docker run --rm -t -v /var/run/docker.sock:/var/run/docker.sock -v "${DOCKER_PATH}":/usr/bin/docker operator-olm-verification
+else
+  docker run --rm -t operator-olm-verification
```

Comment:
is this only for the "skipping" message? if so, it can be moved here and reduce the `if` duplication

Reply:
what do you mean?

Reply:
I mean that we could skip the validation here, instead of later inside

Reply:
why? we still want to run the following command for Power:
`operator-courier --verbose verify deploy/olm-catalog/ibm-block-csi-operator-community`

Reply:
oh, right, then it's not only for the "skipping" message, got it